client/tx/search.go

```diff
@@ -30,24 +30,32 @@ func SearchTxCmd(cdc *codec.Codec) *cobra.Command {
 		Use:   "txs",
 		Short: "Search for all transactions that match the given tags.",
 		Long: strings.TrimSpace(`
-Search for transactions that match the given tags. By default, transactions must match ALL tags
-passed to the --tags option. To match any transaction, use the --any option.
-
-For example:
-
-$ gaiacli tendermint txs --tag test1,test2
-
-will match any transaction tagged with both test1,test2. To match a transaction tagged with either
-test1 or test2, use:
+Search for transactions that match exactly the given tags. For example:
 
-$ gaiacli tendermint txs --tag test1,test2 --any
+$ gaiacli query txs --tags <tag1>:<value1>&<tag2>:<value2>
 `),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			tags := viper.GetStringSlice(flagTags)
+			tagsStr := viper.GetString(flagTags)
+			tagsStr = strings.Trim(tagsStr, "'")
+			var tags []string
+			if strings.Contains(tagsStr, "&") {
+				tags = strings.Split(tagsStr, "&")
+			} else {
+				tags = append(tags, tagsStr)
+			}
 
-			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			var tmTags []string
+			for _, tag := range tags {
+				if !strings.Contains(tag, ":") {
```

Comment:
If the key had a colon, you couldn't search for it. Doesn't seem hugely problematic.

Reply:
Should note this in docs somewhere!

Reply:
done